doc/development.md

```diff
@@ -104,7 +104,7 @@ When an experimental version of the Ignition config spec (e.g.: `2.3.0-experimen
 - Any configs with a `version` field set to the previously experimental version will no longer pass validation. For example, if `2.3.0-experimental` is being marked as stable, any configs written for `2.3.0-experimental` should have their version fields changed to `2.3.0`, for Ignition will no longer accept them.
 - A new experimental spec version will be created. For example, if `2.3.0-experimental` is being marked as stable, a new version of `2.4.0-experimental` will now be accepted, and start to accumulate new changes to the spec.
 - The new stable spec and the new experimental spec will be identical. The new experimental spec is a direct copy of the old experimental spec, and no new changes to the spec have been made yet, so initially the two specs will have the same fields and semantics.
-- The HTTP `user-agent` header that Ignition uses whenever fetching an object and the HTTP `accept` header that Ignition uses whenever fetching a config will be updated to advertise the new stable spec.
+- The HTTP `user-agent` header that Ignition uses whenever fetching an object and the HTTP `accept`, `ID`, `VERSION_ID` (if present), and `VARIENT_ID` (if present) headers that Ignition uses whenever fetching a config will be updated to advertise the new stable spec.
```

Comment:
I don't think those are great header names. Specifically, I think they should:
 * be namespaced to ignition/initramfs/os-release
 * follow the usual HTTP naming convention (dashed camelcase)

So, something along the line of `Ignition-Initramfs-ID` and `Ignition-Initramfs-Version-ID`.

It is also a good idea to document their semantic in a separate doc in this repo, and use that as a reference to register them as [provisional headers](https://tools.ietf.org/html/rfc3864) at IANA, if we expect web servers to react on them.

Reply:
Updated header names. I still need to look into the provisional header stuff.

Reply:
@Rahuls0720 we can leave that for later, it's just paperwork. But I would still replace the remaining underscores with dashes (i.e. `Version-ID` etc.) and capitalize `User-Agent` too.